### PR TITLE
Update Dockerfile.konflux for adding label

### PR DESCRIPTION
**Pull request by MohammadiIram**

### CODE REVIEW

File: Dockerfile.konflux [hunk 1]
```diff
@@ -89,6 +89,7 @@ LABEL com.redhat.component="odh-modelmesh-runtime-adapter-container" \
       maintainer="['managed-open-data-hub@redhat.com']" \
       io.k8s.display-name="odh-model-serving-runtime-adapter" \
       io.k8s.description="odh-model-serving-runtime-adapter" \
+      io.openshift.tags="" \
       com.redhat.license_terms="https://www.redhat.com/licenses/Red_Hat_Standard_EULA_20191108.pdf"
```
Comment (spolti) [inline Dockerfile.konflux:93]: what is the purpose of this empty label?

Comment (MohammadiIram) [reply Dockerfile.konflux:93]: It is required as per konflux labels, not sure what could be the correct value hence kept it as empty. for ref https://github.com/release-engineering/rhtap-ec-policy/blob/main/data/rule_data.yml#L100

Comment (spolti) [reply Dockerfile.konflux:93]: these tags will be present in the image, so having it empty is better to remove it.

@terrytangyuan @Jooho @israel-hdez suggestions on what we ca add to this annotation?

a few suggestions:

- component, e.g. modelmesh , kserve, rest-proxy

```bash
 io.openshift.tags="kserve,openshift-ai,rhoai,model-server,single-model"
```

**Comment by MohammadiIram** [comment]
No more required to add it to dockerfile

**Pull request closed without merging**